webpack/assets/javascripts/foreman_toast_notifications.js

```diff
@@ -0,0 +1,39 @@
+import * as ToastActions from './react_app/redux/actions/toasts';
```

Comment:
notify needs to be accessible from the legacy javascript so I don't think a react component is feasible